core/server/proxy/src/main/java/alluxio/proxy/s3/S3RestServiceHandler.java

```diff
@@ -893,7 +925,9 @@ public Response createObjectOrUploadPart(@HeaderParam("Content-MD5") final Strin
               .setMode(PMode.newBuilder()
                   .setOwnerBits(Bits.ALL)
                   .setGroupBits(Bits.ALL)
-                  .setOtherBits(Bits.NONE).build());
+                  .setOtherBits(Bits.NONE).build())
```

Comment:
I think only the CreateFilePOptions and CreateDirectoryPOptions need this option.